aplt/scenarios.py

```diff
@@ -249,6 +290,80 @@ def _test_multiple_spawn():
     yield spawn("aplt.scenarios:basic, 1, 1, 0")
 
 
+def _api_test():
```

Comment:
should these wrapper scenarios be above the 'TEST SCENARIOS' line with the other scenarios?